src/lib.rs

```diff
@@ -28,13 +31,10 @@ use ecmascript::{
         util::COMMENTS,
     },
 };
-pub use ecmascript::{
+pub use swc_ecmascript::{
```

Comment:
Is this required? I think ecmascript:: would work because `pub use swc_ecmascript as ecmascript` exists

Reply:
you're right, it's not needed